adapter/model.go

```diff
@@ -685,6 +717,15 @@ type ApmDepositPaymentResponse struct {
 	WalletTransaction   *WalletTransaction   `json:"walletTransaction"`
 }
 
+type PayoutAccountResponse struct {
+	Id                  *int64             `json:"id"`
```

Comment:
why this fields are pointer?

Reply:
Yes, most of these fields are not nullable, I will remove pointers.